pandas/src/parser/tokenizer.c

```diff
@@ -1675,23 +1483,20 @@ double xstrtod(const char *str, char **endptr, char decimal,
   num_digits = 0;
   num_decimals = 0;
 
-  // Process string of digits
-  while (isdigit(*p))
-  {
+  // Process string of digits.
```

Comment:
what are you referring? this is just a program that we run so why does it matter if it's updated or not (and 4 months is pretty recent)
how does it matter if it's not in python 3?
it's checking c code